microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -20,145 +20,184 @@ ms.date: 05/04/2021
 
-1. Go to the **Sources** tool.
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
 
-1. In the **Navigator** pane (on the left), select the **Page** tab.
+1. In DevTools, on the main toolbar, select the **Sources** tab.  If that tab isn't visible, click the **More tabs** (![The More tabs icon](../media/more-tabs-icon-light-theme.png)) button.
 
-1. Click **More options** `...` and then clear the **Group by folder** checkbox.
+1. In the **Navigator** pane in the upper left, select the **Page** tab.
 
-   ![The Group by folder option.](../media/resources-sources-page-resource-group-by-folder.msft.png)
+1. Click **More options** (...) and then clear the **Group by folder** option:
 
-    Resources are organized by file type.  Within each file type, the resources are organized alphabetically.
+   ![The Group by folder option in the Page tab of the Sources tool](../media/resources-sources-page-resource-group-by-folder.msft.png)
 
-   ![The Page tab after clearing the Group by folder check mark.](../media/resources-sources-page-resources-empty-not-grouped-by-folder.msft.png)
+   All the resource files for each domain (such as **Top**) are grouped together in a single alphabetized list:
+
+   ![The Page tab in the Sources tool when the 'Group by folder' option is cleared](../media/resources-sources-page-resources-empty-not-grouped-by-folder.msft.png)
 
 
 <!-- ====================================================================== -->
-## Browse resources by file type in the **Application** tool
+## Browse resource files by file type in the Frames tree in the Application tool
 
-To group resources together based on their file type:
+One way to view webpage resources grouped by file type is by using the **Frames** tree in the **Application** tool:
 
-1. Select the **Application** tab.  The **Application** tool opens.  By default, the **Manifest** pane usually opens first.
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
 
-   ![The Application tool.](../media/resources-application-mainfest-airhorner.msft.png)
+1. In DevTools, on the main toolbar, select the **Application** tab.  If that tab isn't visible, click the **More tabs** (![More tabs button](../media/more-tabs-icon-light-theme.png)) button, or else the **More Tools** (![The More Tools icon](../media/more-tools-icon-light-theme.png)) button.
 
-1. Scroll down to the **Frames** pane.
+1. Select the **Application** tab.  The **Application** tool opens and by default shows the **Manifest** pane first:
+   
+   ![The Application tool](../media/resources-application-mainfest-airhorner.msft.png)
```

Comment:
I'm considering deleting this low-value, even misleading screenshot, to increase focus on the png that's below it.  As long as we have this transitional-navigational screenshot:
Add red box on "Application" tab and on "Manifest" node in tree on left.

Reply:
done